subworkflows/local/utils_nfcore_seqinspector_pipeline/main.nf

```diff
@@ -279,8 +279,8 @@ def methodsDescriptionText(mqc_methods_yaml) {
     meta["tool_bibliography"] = ""
 
     // TODO nf-core: Only uncomment below if logic in toolCitationText/toolBibliographyText has been filled!
```

Comment:
Great! You just need to remove this todo too and then I think it's good to go :)